pkg/apis/release/v1alpha1/types.go

```diff
@@ -176,6 +176,9 @@ type ReleasePayloadStatus struct {
 	// Supported conditions include PayloadCreated, PayloadFailed, PayloadAccepted, and PayloadRejected.
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
 
+	// ReleaseCreationJobResult stores the results of the release creation job
```

Comment:
what does the release creation job do?

Reply:
The release creation job is the `batch/v1` job that's responsible for creating/mirroring the actual release itself:
```
oc adm release new --name=4.11.0-0.nightly-2022-04-07-155217 --from-image-stream=4.11-art-latest-2022-04-07-155217 --namespace=ocp --to-image=registry.ci.openshift.org/ocp/release:4.11.0-0.nightly-2022-04-07-155217 --reference-mode=source
```

Reply:
Done

Reply:
Update this comment to say what it does. Does this "build and push a release payload image to the location specified in .spec.PayloadCoordinates ?

Reply:
Done!